dbclient/HiveClient.py

```diff
@@ -300,6 +302,27 @@ def export_hive_metastore(self, cluster_name=None, metastore_dir='metastore/', d
             print("Failed count: " + str(total_failed_entries))
             print("Total Databases attempted export: " + str(len(all_dbs)))
 
+    def get_successful_exported_metastore_tables(self, success_log):
+        tables = set()
```

Comment:
Does this return tables + databases, what's the format we expect here? Fully qualified table names like `dbname.tablename`?

Reply:
done